common/Util.cpp

```diff
@@ -325,6 +325,13 @@ namespace Util
         return std::getenv("DISPLAY") != nullptr;
     }
 
+
+    bool isKitInProcess()
+    {
+        const char* kitInProcess = std::getenv("KIT_IN_PROCESS");
+        return (kitInProcess != nullptr && strcmp(kitInProcess, "true") == 0);
```

Comment:
This can be static. Also, is there a reason to have it set to "true" and "false"? We can simply define the `KIT_IN_PROCESS` for 'true' and not define it for 'false'. And avoid this comparison (just check for `!= nullptr`).

Reply:
@Darshan-upadhyay1110 as you have separate binaries now: regular one and  "single process" one, that env variable is not needed I think.
You should make this true when you link single process code probably.